backend/resources/contact.py

```diff
@@ -1,34 +1,77 @@
-import os
+import logging
+from textwrap import dedent
 
 from http import HTTPStatus
 
 from flask import request
-from flask_mail import Message
+from flask import current_app
 from flask_restful import Resource
 
 from marshmallow import ValidationError
 
 from backend.serializers.message_serializer import MessageSchema
 from backend.extensions import mail
+from backend.helpers import wrap_io
+
+
+MAIL_RECIPIENT = current_app.config["MAIL_RECIPIENT"]
+MAIL_WEB_SERVER = current_app.config["MAIL_WEB_SERVER"]
+BASE_URL = current_app.config["BASE_URL"]
 
 
 class SendMessage(Resource):
     def post(self):
         try:
             message_schema = MessageSchema()
-            new_msg = message_schema.load(request.json)
+            data = message_schema.load(request.json)
         except ValidationError as err:
             return {"message": err.messages}, HTTPStatus.BAD_REQUEST
-        name, phone, content = new_msg["name"], new_msg["phone"], new_msg["content"]
-        base_url = os.environ.get("BASE_URL", "codeforpoznan.pl")
-
-        msg = Message(
-            subject=f"Email z {base_url} od {name}",
-            sender=new_msg["email"],
-            reply_to=new_msg["email"],
-            recipients=[f"hello@{base_url}"],
+
+        name, email = data["name"], data["email"]
+        phone, content = data["phone"], data["content"]
+
+        subject = f"""
+            Email z {BASE_URL} od {name}
+        """
+
+        body = f"""
```

Comment:
bit easier to write that way, we can also invest some time in development of HTML templated emails, but this should be done in separate ticket (we don't have templating library installed and it would be out of scope for this one).